sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/CBDE/CbdeHandler.cs

```diff
@@ -35,6 +35,44 @@
 
 namespace SonarAnalyzer.Rules.CSharp
 {
+    public class ThreadCpuStopWatch
```

Comment:
should this be `private` (and moved to the end of the file)?

Reply:
done